friends/src/FriendsList.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+
+export default class FriendsList extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            friends: []
+        };
+    }
+      
+    componentDidMount() {
+        axios
+            .get('http://localhost:5000/friends')
+            .then(response => {
+                this.setState(() => ({ friends: response.data }));
```

Comment:
Instead of doing another call to the API, you could pass down the already existing data from the app component. Every time you make an API call, it takes time to receive a response, so try to do it a minimal amount of times.